src/languages/en.ts

```diff
@@ -4829,5 +4829,23 @@ export default {
         date: 'Date',
         time: 'Time',
         none: 'None',
+        reasonVisibleInLHN: {
+            hasDraftComment: 'Has draft comment',
+            hasGBR: 'Has GBR',
+            pinnedByUser: 'Pinned by user',
+            isNonReimbursedIOU: 'Is non reimbursed IOU',
```

Comment:
Should this have a hyphen?
```suggestion
            isNonReimbursedIOU: 'Is non-reimbursed IOU',
```